### PR TITLE
Python 2 + compile fixes

### PR DESCRIPTION
**Pull request by ShabbyX**
Needed for ANGLE.

### CODE REVIEW

File: scripts/lvl_genvk.py [hunk 1]
```diff
@@ -21,12 +21,13 @@
 
 def startTimer(timeit):
     global startTime
-    startTime = time.process_time()
```
Comment (mark-lunarg) [inline scripts/lvl_genvk.py]: @ShabbyX, this is part of the [vulkan registry which is upstream of this repo](https://gitlab.khronos.org/vulkan/vulkan).  We'd suggest requesting the changes there, and they'll filter back down to us.

Comment (ShabbyX) [reply scripts/lvl_genvk.py]: I seem to be unable to fork the repo on Khronos' gitlab (on gitlab itself). Do you think you would be able to commit this simple change on my behalf?

Comment (mark-lunarg) [reply scripts/lvl_genvk.py]: Sorry, Vulkan/Vulkan is the private repo.  The downstream public version is [Vulkan-Docs](https://github.com/KhronosGroup/Vulkan-Docs).  See if that's better.

Comment (ShabbyX) [reply scripts/lvl_genvk.py]: Done. Vulkan-Docs#894

Comment (ShabbyX) [reply scripts/lvl_genvk.py]: And removed the changes to this script from the PR.

**Comment by mark-lunarg** [comment]
Thanks again!

**Pull request merged**